### DISCUSSION TITLE
BadgeText

### DISCUSSION THREAD

**Discussion by angelru** [xamarin/XamarinCommunityToolkit#1214] [Q&A]
`TabViewItem BadgeText="0"`

not hide the badge?

**Comment by AndreiMisiukevich** [comment]
I guess it hides now.
Doesn't it work?

**Reply by angelru** [reply]
the one of the TabViewItem no, the one of BadgeView yes. Also if I have a ControlTemplate in TabViewItem, BadgeText is not displayed, BadgeText borders are not working for me either. All this on Android 9.

Example:
[App1.zip](https://github.com/xamarin/XamarinCommunityToolkit/files/6348324/App1.zip)

**Reply by angelru** [reply]
ping @AndreiMisiukevich

**Reply by AndreiMisiukevich** [reply]
Ah, sorry. I messed TabViewItem and BadgeView.

I don't know, maybe it's a bug

CC @jsuarezruiz
